index.src.html

```diff
@@ -332,6 +332,32 @@ <h3 dfn export id="referrer-policy-origin" oldids="referrer-policy-state-origin"
     priori</em> authenticated URLs</a>.
   </div>
 
+  <h3 dfn export id="referrer-policy-safe-origin" oldids="referrer-policy-state-safe-origin">"<code>safe-origin</code>"</h3>
+
+  The <a>"<code>safe-origin</code>"</a> policy sends the
+  <a lt="ASCII serialization of an origin">ASCII serialization</a> of the
+  <a>origin</a> of the <a>request client</a> when making requests from
+  <a>TLS-protected</a> <a>environment settings object</a> to a
+  <a><em>a priori</em> authenticated URL</a>, and requests from
+  <a>request clients</a> which are <em>not</em> <a>TLS-protected</a> to any
+  <a>origin</a>.
+
+  Requests from <a>TLS-protected</a> <a>request clients</a> to non-<a><em>a
+  priori</em> authenticated URLs</a>, on the other hand, will contain no
+  referrer information. A <code><a>Referer</a></code> HTTP header will not be
+  sent.
+
```

Comment:
> Ok, is the following a correct summary of `safe-origin`?
> - same-origin requests always include the full URL as the referrer

No. Just like with `origin`, same-origin requests don't get the path.

> - https -> http requests have no referrer

Correct. That's the only difference when compared to `origin`.

> - https -> https requests (cross-origin) have the origin as the referrer
> - http -> http/https (cross-origin) have the origin as the referrer

Correct.